### DISCUSSION TITLE
Clarification regarding infeasibility

### DISCUSSION THREAD

**Discussion by tripiit16** [yalmip/YALMIP#1005] [General discussion] [answered]
Dear Sir,
PFA is the code for three-phase OPF and the corresponding equations. The result is unbounded infeasible. I have tried debugging the infeasibility using the link provided by you. But still, the result is the 
[doubt.txt](https://github.com/yalmip/YALMIP/files/7235891/doubt.txt)
![unnamed](https://user-images.githubusercontent.com/68984517/134902802-99ee288a-7590-484b-a401-cbd899c63679.png)
same.

**Comment by johanlofberg** [comment, marked as answer]
You don't seem to have done any real effort in finding the problem, as I still easily can cut out major parts of the code and still have infeasibility. Your code is weird also as you call some stuff con (which never is used) and then use another one const (which is used)

`con=[con,pgrid(3,1)==real(Load(1,3))+(aux(1,:)*pij(:,9))];

% const=[const,qgrid(1,1)==imag(Load(1,1))+(aux(1,:)*qij(:,1))];
% const=[const,qgrid(2,1)==imag(Load(1,2))+(aux(1,:)*qij(:,5))];
con=[con,qgrid(3,1)==imag(Load(1,3))+(aux(1,:)*qij(:,9))];

% const=[const, aux(2:nb,:)*(pij(:,1))+real(Load(2:nb,1))-Pg(1,2:nb)'==0];
% const=[const, aux(2:nb,:)*(pij(:,5))+real(Load(2:nb,2))-Pg(2,2:nb)'==0];
con=[con, aux(2:nb,:)*(pij(:,9))+real(Load(2:nb,3))-Pg(3,2:nb)'==0];

const=[const, aux(2:nb,:)*(qij(:,1))+imag(Load(2:nb,1))==0];
const=[const, aux(2:nb,:)*(qij(:,5))+imag(Load(2:nb,2))==0];
const=[const, aux(2:nb,:)*(qij(:,9))+imag(Load(2:nb,3))==0];

const=[const,pij(:,2)==a*(pij(:,1))];
const=[const,pij(:,3)==a*(pij(:,1))];
const=[const,pij(:,4)==a*(pij(:,5))];
const=[const,pij(:,6)==a*(pij(:,5))];
const=[const,pij(:,8)==a*(pij(:,9))];
const=[const,pij(:,7)==a*(pij(:,9))];
const=[const,qij(:,2)==a*(qij(:,1))];
const=[const,qij(:,3)==a*(qij(:,1))];
const=[const,qij(:,4)==a*(qij(:,5))];
% const=[const,qij(:,6)==a*(qij(:,5))];
% const=[const,qij(:,8)==a*(qij(:,9))];
% const=[const,qij(:,7)==a*(qij(:,9))];

% const=[const,-20<=pij<=20];
% const=[const,-20<=qij<=20];
% const=[const,(0<=Pg<=10)];
% const=[const,0<=pgrid<=40];
% const=[const,(0<=qgrid<=40)];

% for i=1:phij
%     const=[const,v(1,Linedata(:,2))==v(1,Linedata(:,1))-2*(sum((r(:,i).*pij(:,i))+(x(:,i).*qij(:,i)),2))'];
%     const=[const,v(2,Linedata(:,2))==v(2,Linedata(:,1))-2*(sum((r(:,i).*pij(:,i))+(x(:,i).*qij(:,i)),2))'];
%     const=[const,v(3,Linedata(:,2))==v(3,Linedata(:,1))-2*(sum((r(:,i).*pij(:,i))+(x(:,i).*qij(:,i)),2))'];
% end

%const = [const, (Vmin)^2<= v(:,:)<=(Vmax)^2];
%const=[const,v(:,1)== 4.160/sqrt(3)];`
